speechtotextv1/speech_to_text_v1.go

```diff
@@ -4796,6 +4796,28 @@ type RecognitionJobs struct {
 	Recognitions []RecognitionJob `json:"recognitions" validate:"required"`
 }
 
+// AudioProperties : AudioProperties struct (Websocket Only)
+type AudioProperties struct {
```

Comment:
Do we really need this struct? The struct was added in python as it didn't have continuous streaming for a microphone example. We dont have the microphone example here, so we can focus on a simple websocket connection where we send the continuous stream.

Reply:
For some of my current work, we would need the SDK to support continuous streaming from a microphone. If this struct is removed, are there any suitable alternatives to enable microphone support?